automatic/ffmpeg/ffmpeg.nuspec

```diff
@@ -30,6 +30,9 @@ Security is a high priority and code review is always done with security in mind
     <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/automatic/ffmpeg</packageSourceUrl>
     <projectSourceUrl>https://git.ffmpeg.org/ffmpeg.git</projectSourceUrl>
     <docsUrl>https://ffmpeg.org/ffmpeg.html</docsUrl>
+    <dependencies>
+      <dependency id="chocolatey" version="0.10.7" />     
```

Comment:
Shouldn't the version be set to `0.10.5`?
From my understanding, dependency on chocolatey should be the minimum version is will work with (I'm saying 0.10.5 because 0.10.4 was broken).